include/bluetooth/scan.rst

```diff
@@ -98,6 +98,31 @@ Check the following table for the details on the behavior of the available filte
 |              | If not all of these types match, the ``not found`` callback is triggered.                                 |
 +--------------+-----------------------------------------------------------------------------------------------------------+
 
+Connection attempts filter
+==========================
+
+After a scanning is started by the application some peripheral devices can disconnect immediately for some reasons during ongoing scanning session.
+This may result in a loop of the connection and disconnection events.
+To avoid this loop you can enable the Connection attempts filter which limits number of the connection tries.
+Use the :option:`CONFIG_BT_SCAN_CONNECTION_ATTEMPTS_FILTER` to enable this filter.
+This filter automatically tracks connected devices and count all disconnection events for given devices if disconnection counts is greater than number of allowed attempts the scanning module ignores this device.
```

Comment:
for given devices. If disconnection count is ...

Reply:
Actually it should be:  `the disconnection count is greater than or equal to`. You can also state: `if the connection attempt count is greater than` :)